Will Heikoff Documentation

```diff
@@ -248,6 +251,79 @@ Working with Jakes code for transforming the employment bin data in the zip code
 	
 7/20/18
 Able to rewrite Jake’s code to be compatible with R and create a table which shows the adjusted buckets, zip codes, naics, 
-	number of establishments, and total manufa. Employment
+	number of establishments, and total manufa. Employment, saved file as ZIPs_Tot_Emp.R in
+	G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Complete Zip Code Industry Detail File folder
 Troubleshooting method to export data from R
 
+ZIPs_Tot_Emp.R file:
```

Comment:
this looks like you pasted the R file into the documentation file. is the correct? If so, please take it back out and upload the R file separately. Then, whenever changes are made in the future, you can just make commits on that file directly and they can be tracked separately from this file.